docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -27,6 +27,11 @@
                 <li><a href="{{BASE_PATH}}/quickstart/tutorial.html">Tutorial</a></li>
                 <li role="separator" class="divider"></li>
                 <li class="title"><span><b>Basic Feature Guide</b><span></li>
+                <!-- li><span><b>Tutorial</b><span></li -->
+                <li><a href="{{BASE_PATH}}/tutorial/tutorial.html">Tutorial</a></li>
```

Comment:
Is there any particular reason to add `tutorial.html` again? As you can see in the above, already we have 
```
<li><a href="{{BASE_PATH}}/quickstart/tutorial.html">Tutorial</a></li>
```

When I build this in my local, the dropdown menu is shown like the below.
<img width="554" alt="screen shot 2016-07-16 at 4 23 15 pm" src="https://cloud.githubusercontent.com/assets/10060731/16893406/a3a3f65e-4b71-11e6-940f-5b8050eb4037.png">